Assets/MRTK/Core/Utilities/WindowsDevicePortal/DevicePortal.cs

```diff
@@ -26,6 +36,28 @@ private enum AppInstallStatus
             InstallFail
         }
 
+        /// <summary>
+        /// Custom certificate handler for device portal request.
+        /// The device portal on HoloLens uses a self-signed certificate, therefore SSL Unity WebRequest will fail.
+        /// As a fix we simply accept all certificates, including self-signed, without further checking, 
+        /// as they do not chain up to any Microsoft Root Certificate.
+        /// </summary>
+        private class BlankCertificateHandler : CertificateHandler
+        {
+            protected override bool ValidateCertificate(byte[] certificateData)
+            {
+                // Accept all the certificates!
+                return true;
+            }
+        }
+
+        private static readonly BlankCertificateHandler BlankCertificateHandlerInstance = new BlankCertificateHandler();
+
+        private static CertificateHandler DevicePortalCertificateHandler
+        {
+            get { return (!VerifySSLCertificates) ? BlankCertificateHandlerInstance : null; }
```

Comment:
```suggestion
            get { return !VerifySSLCertificates ? BlankCertificateHandlerInstance : null; }
```

Reply:
Nit: The extra scope here doesn't add a ton (i.e. makes more sense when there are multiple clauses)

Reply:
Done